### PR TITLE
Change log level jsonrpc4j

### PR DESCRIPTION
**Pull request by patogallaiovlabs**
- Improvement in log level, no need to log so deep.

### CODE REVIEW

File: rskj-core/src/main/java/co/rsk/rpc/netty/JsonRpcWeb3ServerHandler.java [hunk 1]
```diff
@@ -59,9 +59,8 @@ protected void channelRead0(ChannelHandlerContext ctx, ByteBufHolder request) th
         int responseCode;
         try (ByteBufOutputStream os = new ByteBufOutputStream(responseContent);
              ByteBufInputStream is = new ByteBufInputStream(request.content().retain())){
-
             responseCode = jsonRpcServer.handleRequest(is, os);
-        } catch (Exception e) {
+        } catch (Throwable e) {
```
Comment (ajlopezrsk) [inline rskj-core/src/main/java/co/rsk/rpc/netty/JsonRpcWeb3ServerHandler.java:63]: Any reason to this change?

Comment (patogallaiovlabs) [reply rskj-core/src/main/java/co/rsk/rpc/netty/JsonRpcWeb3ServerHandler.java:63]: To catch stackoverflow... but i was thinking that maybe has no sense.

File: rskj-core/src/main/resources/logback.xml [hunk 2]
```diff
@@ -85,6 +88,9 @@
     <logger name="co.rsk.db.migration.OrchidToUnitrieMigrator" level="INFO"/>
     <logger name="co.rsk.db.migration.MissingOrchidStorageKeysProvider" level="INFO"/>
 
+    <!-- Use INFO or upper levels for productive environments. -->
+    <logger name="com.googlecode.jsonrpc4j" level="INFO"></logger>
```
Comment (Vovchyk) [inline rskj-core/src/main/resources/logback.xml:92]: what is the reason for this change?

**Comment by patogallaiovlabs** [comment]
pipeline:run

**Comment by patogallaiovlabs** [comment]
pipeline:run

**Comment by patogallaiovlabs** [comment]
pipeline:run

**Comment by patogallaiovlabs** [comment]
pipeline:run

**Comment by patogallaiovlabs** [comment]
pipeline:run

**Comment by patogallaiovlabs** [comment]
pipeline:run

**Review by ajlopezrsk — DISMISSED** [review]

**Comment by patogallaiovlabs** [comment]
pipeline:run

**Comment by patogallaiovlabs** [comment]
pipeline:run

**Review by ajlopezrsk — APPROVED** [review]
